healthScore/healthScore/views.py

```diff
@@ -19,27 +19,29 @@ def test_default_values(request):
 
 def view_health_history(request):
     if request.method == "GET":
+        return render(request, 'view_history.html')
+    else:
         # Filtering to just userID=5 to simulate it being a users view.
         history_list = healthRecord.objects.filter(userID=5)
 
-        appointment_name = request.GET.get("appointment_name")
+        appointment_name = request.POST.get("appointment_name")
```

Comment:
Please change the request method from POST to GET here as well. And also in the below lines. We can have the rest of the logic inside the 
`if request.method == "GET":` condition itself